packages/apollo-server-micro/README.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Micro
+description: Setting up Apollo Server with Micro
+---
+
+[![npm version](https://badge.fury.io/js/apollo-server-core.svg)](https://badge.fury.io/js/apollo-server-core) [![Build Status](https://circleci.com/gh/apollographql/apollo-cache-control-js.svg?style=svg)](https://circleci.com/gh/apollographql/apollo-cache-control-js) [![Coverage Status](https://coveralls.io/repos/github/apollographql/apollo-server/badge.svg?branch=master)](https://coveralls.io/github/apollographql/apollo-server?branch=master) [![Get on Slack](https://img.shields.io/badge/slack-join-orange.svg)](https://www.apollographql.com/#slack)
+
+This is the [Micro](https://github.com/zeit/micro) integration for the Apollo community GraphQL Server. [Read the docs.](https://www.apollographql.com/docs/apollo-server/) [Read the CHANGELOG.](https://github.com/apollographql/apollo-server/blob/master/CHANGELOG.md)
+
+## Basic GraphQL Microservice
+
+This example demonstrates how to setup a simple microservice using Micro, that
+handles incoming GraphQL requests via the default `/graphql` endpoint.
+
+1) Package installation.
+
+```sh
+npm install --save micro apollo-server-micro
+```
+
+2) `index.js`
+
+```js
+const { ApolloServer, gql } = require('apollo-server-micro');
+
+const typeDefs = gql`
+  type Query {
+    sayHello: String
+  }
+`;
+
+const resolvers = {
+  Query: {
+    sayHello(root, args, context) {
+      return 'Hello World!';
+    },
+  },
+};
+
+const apolloServer = new ApolloServer({ typeDefs, resolvers });
+module.exports = apolloServer.graphqlHandler();
+```
+
+3) `package.json`
+
+```json
+{
+  "main": "index.js",
+  "scripts": {
+    "start": "micro"
+  }
+}
+```
+
+4) After an `npm start`, access `http://localhost:3000` to run queries using
+[`graphql-playground`](https://github.com/prismagraphql/graphql-playground),
+or send GraphQL requests directly to `http://localhost:3000/graphql`.
+
+## CORS Example
+
+Adjust the `Basic GraphQL Microservice` example as follows, to use [`micro-cors`](https://github.com/possibilities/micro-cors):
+
+```sh
+npm install --save micro-cors
+```
+
+```js
+const cors = require('micro-cors')();
+...
+module.exports = cors(apolloServer.graphqlHandler());
+```
+
+## Custom GraphQL Path Example
+
+Adjust the `Basic GraphQL Microservice` example as follows:
+
+```js
+...
+module.exports = cors(apolloServer.graphqlHandler({ path: '/data' }));
```

Comment:
We should leave out the cors option here, since using the previous example. It might also be nice to link to the #basic-graphql-microservice example above when we mention it, in case forgetful readers(like me) didn't check the title 😊

Reply:
Ah I see, this tutorial is cumulative. I'd slightly lean towards standalone sections, since users can pull from the grab bag without something else sharp poking them. Though I'm happy to see it as a cumulative deal if you prefer. If it is building, I'm worried that people will get confused by the headers at the same level.

Reply:
Good point - I'll make each section standalone. Thanks!